src/main/java/edu/ucsb/cs156/frontiers/controllers/CoursesController.java

```diff
@@ -191,4 +196,33 @@ public List<Map<String, Object>> lookUpStudentCourseRoster()
         
         return matchedCourses;
     }
-}
+
+    @Operation(summary = "Student can see what courses they appear on staff roster of and their status in each")
+    @PreAuthorize("hasRole('ROLE_USER')")
+    @GetMapping("/staff")
+    public List<Map<String, Object>> lookUpStaffCourseRoster() 
+    {
+        CurrentUser currentUser = getCurrentUser();
+        String email = currentUser.getUser().getEmail(); 
+        List<CourseStaff> staffRoster = courseStaffRepository.findAllByEmail(email); 
+
+        List<Map<String, Object>> matchedCourses = new ArrayList<>();
+
+        for (CourseStaff st : staffRoster) 
+        {
+            Course course = st.getCourse(); 
+            
+            Map<String, Object> courseInfo = new HashMap<>();
+            courseInfo.put("id", course.getId());
+            courseInfo.put("orgName", course.getOrgName());
+            courseInfo.put("courseName", course.getCourseName());
+            courseInfo.put("term", course.getTerm());
+            courseInfo.put("school", course.getSchool());
+            courseInfo.put("installationId", course.getInstallationId());
+            courseInfo.put("status", st.getOrgStatus()); 
+
+            matchedCourses.add(courseInfo);
```

Comment:
A custom DTO object would be better here.  But I'll take it for now, since it's so late in the game.